src/components/snackbar/snackbar.tsx

```diff
@@ -118,9 +120,7 @@ export class Snackbar {
                         class="mdc-snackbar__label"
                         role="status"
                         aria-live="polite"
-                    >
-                        {this.message}
```

Comment:
Note: This needed to be removed to avoid having the message rendered TWICE on the first call to show(). Instead of _rendering_ the message, we use MDCSnackbar properties to _set_ it before show, just like we do with timeout.